app/metrics.yaml

```diff
@@ -732,4 +744,83 @@ error_page:
     - https://github.com/mozilla-mobile/fenix/pull/2491#issuecomment-492414486
     notification_emails:
     - fenix-core@mozilla.com
+    expires: "2020-03-01"
+
+sync:
```

Comment:
I think we'll want to split this up into a account.auth flow and an account.settings flow. because

```
opened:
    type: event
    description: >
      A user opened the sync page
```

This event is really means opened the page to start the sign in flow to firefox account.